src/m365/spfx/commands/spfx-doctor.ts

```diff
@@ -438,6 +438,21 @@ class SpfxDoctorCommand extends AnonymousCommand {
         range: '^4',
         fix: 'npm i -g yo@4'
       }
+    },
+    '1.16.0-beta.1': {
+      gulp: {
+        range: '^4',
+        fix: 'npm i -g gulp@4'
+      },
+      node: {
+        range: '^12.13 || ^14.15 || ^16.13',
+        fix: 'Install Node.js v12.13, v14.15, v16.13 or higher'
```

Comment:
If I'm reading the MS docs correctly for 1.16.0.beta.1, it mentions that it won't support NodeJS v12 and v14 anymore. [The article](https://docs.microsoft.com/sharepoint/dev/spfx/release-1.16#remove-nodejs-12-and-14-support)

Currently, projects build with the latest beta version run perfectly in Node v12 and v14. Should we leave these versions in the doctor command or follow the docs and remove them? @waldekmastykarz what do you think?

Reply:
Let's align with docs as they represent the supported and intended versions. It could be that v12 works, but there could be edge cases because it's not fully tested.

Reply:
@milanholemans can you align the NodeJS versions to the docs 😄

Reply:
Marked as draft to indicate we need a change before we merge it

Reply:
In the docs they claim indeed there is no support for Node v12 and v14. However, I usually check the microsoft node modules and there I saw this:

![image](https://user-images.githubusercontent.com/11723921/188172659-e93320ae-cc6a-4df0-a2bb-72073b63dc16.png)

I've tested it on Node v14.20.0 and it works just fine. Not sure what to do with this 😃.

Reply:
In package.json of the generator the engines.node property is set to v16 if I'm not mistaken. Again, I'd err on the side of docs and what the PG intended. Since it's a preview version, it could be that not all of the code has been updated yet. When in doubt, we can always triple check with @AJIXuMuK

Reply:
Should I just update the PR so only Node v16.13+ is supported? However in the docs they are only saying "Node v16".
Seems like we're not getting a response of @AJIXuMuK (subtile 2nd ping 😉)

Reply:
Yeah, it can be hard to get response from me on Saturday ;)
With 1.16 we're going to support node v16 only. I'd just copy the value of `engine` from the generated project.